docs/tutorials/join-dal-baker/run-baker.md

```diff
@@ -124,6 +124,13 @@ However, make sure not to run the baker twice **for the same protocol version an
 
 You can upgrade accusers with a similar process.
 
+## Backing up and restoring the baker
+
+The Octez baking daemon stores persistent operational data in the Octez client's data directory, notably consensus high-water marks and [random seed nonces](https://tezos.gitlab.io/active/randomness_generation.html).
+If you want to back up the baker or move it to another machine and restore it, you must copy the nonce file or files from the Octez client's data directory to the equivalent directory on the new machine.
+These nonce files are named `net<NETWORK_ID>_stateful_nonces` and `net<NETWORK_ID>_nonces`, where `<NETWORK_ID>` is the ID of the network, such as `netXdQprcVkpaWU_stateful_nonces` for Mainnet or `netXnHfVqm9ie_stateful_nonces` for Ghostnet.
+All systems have the `net<NETWORK_ID>_stateful_nonces` file but only legacy baking deployments have the `net<NETWORK_ID>_nonces` file.
```

Comment:
```suggestion
All systems have the `net<NETWORK_ID>_stateful_nonces` file but only legacy baking deployments have the `net<NETWORK_ID>_nonces` file.
```
systems -> deployments. 

We should expand  legacy to mean, that were first deployed with Octez versions earlier/older than v20.0~rc1